### PR TITLE
Add spell effects for several Psychic cantrips

### PR DESCRIPTION
**Pull request by TikaelSol**
Plus bonus fix for Telekinetic Rend's damage.

### CODE REVIEW

File: packs/data/spells.db/telekinetic-rend.json [hunk 1]
```diff
@@ -67,6 +45,61 @@
...
+                                    "value": "slashing"
+                                },
+                                "value": "1d6"
+                            }
+                        }
+                    },
+                    "heightening": {
+                        "damage": {
+                            "VSfHXFSJQGTzmS13": "1d6",
+                            "fxdjllexjn5g19ig": "1d8"
```
Comment (TikaelSol) [inline packs/data/spells.db/telekinetic-rend.json:97]:
```suggestion
                            "VSfHXFSJQGTzmS13": "1d6"
```
Something is weird about the spell variants, that ID doesn't even exist on the item.

File: packs/data/spell-effects.db/spell-effect-ghostly-shift.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            "value": 3
+        },
+        "rules": [
+            {
+                "exceptions": [
+                    "force"
+                ],
+                "key": "Resistance",
+                "type": "all-damage",
+                "value": "floor(@item.level/2)"
```
Comment (simonward) [inline packs/data/spell-effects.db/spell-effect-ghostly-shift.json:25]: Should this be ceil?

Comment (TikaelSol) [reply packs/data/spell-effects.db/spell-effect-ghostly-shift.json:25]: It's a level 3 cantrip, so level 3 = 2 resist, level 5 = 3, etc.

Comment (TikaelSol) [inline packs/data/spell-effects.db/spell-effect-ghostly-shift.json:25]:
```suggestion
                "value": "ceil(@item.level/2)"
```

**Pull request merged**